samples/drivers/touch/README.rst

```diff
@@ -0,0 +1,31 @@
+.. zephyr:code-sample:: touch
+   :name: touch
+
+   Basic touchscreen sample.
+
+Overview
+********
+This sample will draw a small plus in the last touched coordinates, that way you can check
+if the touch screen works for a board, examine its parameters such as inverted/swapped axes.
+
+Building and Running
+********************
+While this is a generic sample and it should work with any boards with both display controllers
+and touch controllers supported by Zephyr (provided the corresponding /chosen node properties
+are set i.e. `zephyr,touch` and `zephyr,display`).
+Below is an example on how to build the sample for :ref:`stm32f746g_disco_board`;
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/touch
```

Comment:
please move the sample to `samples/subsys/input/draw_touch_events` (happy to hear about other suggestions, but finding a better name than the generic "touch" is important, and so is moving to a either input or display subsys)

Reply:
Moved